UnitTests/ControllerTests/CreationControllerTests/GetVisualizationRulesAsyncActionTests.cs

```diff
@@ -330,11 +330,12 @@ public async Task GetVisualizationRulesAsync_VerticalBarChart_AllowsShowingDataP
         [TestCase(VisualizationType.PercentVerticalBarChart, 1, 2, 5, 1, 1)]
         [TestCase(VisualizationType.HorizontalBarChart, 1, 1, 5, 1, 1)]
         [TestCase(VisualizationType.GroupHorizontalBarChart, 1, 2, 2, 1, 1)]
-        [TestCase(VisualizationType.StackedHorizontalBarChart, 1, 1, 3, 3, 1)]
-        [TestCase(VisualizationType.PercentHorizontalBarChart, 1, 1, 3, 3, 1)]
+        [TestCase(VisualizationType.StackedHorizontalBarChart, 1, 1, 3, 1, 3)]
+        [TestCase(VisualizationType.PercentHorizontalBarChart, 1, 1, 3, 1, 3)]
```

Comment:
Why have these changed?

Reply:
I had to change the test to get coverage for pyramid chart settings as it didn't work without ordinal variable type. This broke these two test cases because variable types changed in a way that didn't support them so I had to change them